### ISSUE TITLE
Option cmyk=True does not work in export()

### ISSUE THREAD

**Issue by dreerr** [plotdevice/plotdevice#38]
When running 

```
export("test.pdf", cmyk=True)
```

an exception is raised:

```
NameError: global name 'CMYK' is not defined
```

Version 0.10.0 (r964)

**Comment by dimitre** [comment]
+1 to this issue
exporting with command line --cmyk parameter works but generates an RGB pdf file too

**Comment by samizdatco** [comment]
fixed in 59ee83adc27d90a9c962797253b2e2b7233d2282
